pkg/chains/constants.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2020 The Tekton Authors
```

Comment:
New files should probably have the current year's in the copyright header. Please adjust in the other new files as well.

Reply:
Done